lib/src/shared/message.dart

```diff
@@ -69,8 +71,8 @@ StreamTransformer<GrpcMessage, GrpcMessage> grpcDecompressor() =>
         handleData: (GrpcMessage value, EventSink<GrpcMessage> sink) {
       if (value is GrpcData) {
         if (value.isCompressed) {
-          // TODO(dart-lang/grpc-dart#6): Actually handle decompression.
-          sink.add(GrpcData(value.data, isCompressed: false));
+          final decompressedData = GZipDecoder().decodeBytes(value.data);
```

Comment:
Can we take the decompression part out of this PR?

Even though this is awesome I am reluctant to take a dependency on `package:archive`.

Reply:
done